### PR TITLE
Minor modifications to make the plugin work

### PR DESCRIPTION
**Pull request by hadi-el-hajj**

### CODE REVIEW

File: src/ios/CameraStream.swift [hunk 1]
```diff
@@ -49,30 +50,33 @@ class CameraStream: CDVPlugin, AVCaptureVideoDataOutputSampleBufferDelegate {
...
-                // lets start some session baby :)
                 session!.startRunning()
             }
         }
     }
     
     @objc(pause:)
     func pause(command: CDVInvokedUrlCommand){
-        if session?.isRunning {
+        if ((session?.isRunning) != nil) {
```
Comment (pirumpi) [inline src/ios/CameraStream.swift:60]: Are you checking for the possibility that session is not null but isRunning could be?

File: src/ios/CameraStream.swift [hunk 2]
```diff
@@ -49,30 +50,33 @@ class CameraStream: CDVPlugin, AVCaptureVideoDataOutputSampleBufferDelegate {
...
             session?.stopRunning()
         }
     }
     
     @objc(resume:)
     func resume(command: CDVInvokedUrlCommand){
-        if session?.isRunning {
-            return
+        if ((session?.isRunning) != nil) {
```
Comment (pirumpi) [inline src/ios/CameraStream.swift:67]: The question mark is doing the null check, do you really need the second verification?

Comment (hadi-el-hajj) [reply src/ios/CameraStream.swift:67]: Both of these modifications (this one and the one you replied to above) were in fix of this error I have got when trying to run the plugin: 
**Optional type 'Bool?' cannot be used as a boolean; test for '!= nil' instead**
The change I applied was the change proposed by XCode, which should be equivalent to session?.isRunning

Comment (hadi-el-hajj) [reply src/ios/CameraStream.swift:67]: Capture d’écran 2021-08-03 à 14 45 48

**Pull request merged**